packages/vite/src/node/server/middlewares/proxy.ts

```diff
@@ -116,3 +115,10 @@ export function proxyMiddleware(
     next()
   }
 }
+
+function testUrl(url: string, target: string) {
```

Comment:
"test" seems a bit ambiguous here or maybe doesn't describe what this function does that clearly. Is this test as in `RegExp.test`? Or are these URLs used for testing? Maybe there could be a better name for this function that describes what it's checking for?

Reply:
thanks for the quick review, is `doesProxyContextMatchUrl` ok? https://github.com/vitejs/vite/pull/4893/commits/437457df4ab2d7d86b95e662857a036b425e377b